scripts/version

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+releaseVersion="v${npm_package_version} ($(date +%Y-%m-%d))"
+
+sed -i "" "s/Unreleased/$releaseVersion/" CHANGELOG.md
```

Comment:
Should we ignore the capital letter for `Unreleased`?

Or can we automate adding unreleased to the top of the changelog after as well?

Reply:
i suppose we should ignore caps in general, maybe

```suggestion
sed -i "" "s/unreleased/$releaseVersion/I" CHANGELOG.md
```

Either way, I don't love automatically re-adding "Unreleased" because I think changelog readability suffers. For example:

> # Restricted Input - Release Notes
> 
> ## Unreleased
> 
> ## 2.0.1
> 
> * Fix bug where input would be formatted when using `setPattern` even if input is empty

To me this is confusing because at first glance it's unclear if it's saying 2.0.1 is unreleased? Sure after I think about it for a second it's clear that's not the case, but I don't want to make anyone think more than they already have to.

Reply:
oh jk BSD `sed` doesn't have a flag to ignore case apparently.

```suggestion
sed -i "" "s/[Uu]nreleased/$releaseVersion/" CHANGELOG.md
```